playground/backend/configs/SDK_PYTHON.json

```diff
@@ -0,0 +1,6 @@
+{
+  "compile_cmd": "",
+  "run_cmd": "python3",
```

Comment:
ah this is interesting - now that I think about this, we should mention in the README that to update runtime environments, we need to change these config files (we cna leave it for later)